app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,35 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
mmm, then you don't need it in dishes_controller? I would move it to application-controller!